test/cli/help/remote-create-table/expected-stdout

```diff
@@ -34,6 +34,7 @@ Flags:
       --columns strings            comma-separated list of column names
       --columns-from string        the path to the table definition file in json
       --name string                name of the table (required if the tableId argument is empty)
+      --options-from string        the path to the set of specific settings for bigquery
```

Comment:
Suggestions:

`--columns-from string        the path to the table definition file in json` 
- :arrow_right: `--columns-from string        the path to the columns definition file in json`


-------------------------

`--options-from string        the path to the set of specific settings for bigquery` 
- :arrow_right: `--options-from string        the path to the table definition file with backed specific options`